web/styles/message_view_header.css

```diff
@@ -119,4 +119,22 @@
             vertical-align: middle;
         }
     }
+
+    .narrow_description_extended {
+        word-break: break-word;
```

Comment:
Have you experimented with [hyphenation](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens)? There can be some unfortunate breaks and orphaned punctuation with this approach. For example, note the period bumped to the line following _development_:

![Screenshot 2024-12-18 at 12 39 15 PM](https://github.com/user-attachments/assets/7e18c3c6-13a9-4328-89f2-81d88059727f)